src/test/java/org/fundacionjala/coding/jorge/VowelsTest.java

```diff
@@ -1,4 +1,4 @@
-package org.fundacionjala.coding.nestor;
+package main.java.org.fundacionjala.coding.jorge;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') This file "VowelsTest.java" should be located in "main/java/org/fundacionjala/coding/jorge" directory, not in "/home/travis/build/AT-08/coding/src/test/java/org/fundacionjala/coding/jorge". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1598)